applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -41,6 +46,33 @@ def __init__(self, Model, settings ):
     def ExecuteFinalizeSolutionStep(self):
         KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','COMPUTE LIFT')
 
+        self.__CalculateWakeTangentAndNormalDirections()
```

Comment:
Can we make the methods protected instead of private? So they can be called by derived classes (embedded_lift_process.py)
```suggestion
        self._CalculateWakeTangentAndNormalDirections()
```